build.sbt

```diff
@@ -65,6 +65,21 @@ scalaJSStage in ThisBuild := (sys.props.get("scalaJSStage") match {
   case _ => FastOptStage
 })
 
+val javacSettings = Seq(
+  "-source", "1.8",
+  "-target", "1.8",
```

Comment:
This needs to be scoped in `javacOptions in (Compile, compile)` because otherwise `doc` (which creates javadoc) will pass it to `javadoc` which doesn't support it (sbt/sbt#1785).

Reply:
https://github.com/playframework/play-json/pull/277